multichain-testing/test/fast-usdc/config.ts

```diff
@@ -0,0 +1,27 @@
+import type { IBCChannelID } from '@agoric/vats';
+
+export const oracleMnemonics = {
+  oracle1:
+    'cause eight cattle slot course mail more aware vapor slab hobby match',
+  oracle2:
+    'flower salute inspire label latin cattle believe sausage match total bless refuse',
+  oracle3:
+    'surge magnet typical drive cement artist stay latin chief obey word always',
+};
+harden(oracleMnemonics);
+
+export const makeFeedPolicy = (nobleAgoricChannelId: IBCChannelID) => {
+  return JSON.stringify({
+    nobleAgoricChannelId,
+    nobleDomainId: 4,
+    chainPolicies: {
+      Arbitrum: {
+        cctpTokenMessengerAddress: '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
+        chainId: 42161,
+        confirmations: 2,
+        nobleContractAddress: '0x19330d10D9Cc8751218eaf51E8885D058642E08A',
```

Comment:
are those arbitrary values chosen by us?

Or is there an external design constraint here? If so, let's please cite it.

Reply:
For context: https://developers.circle.com/stablecoins/evm-smart-contracts

That's for `cctpTokenMessengerAddress`, but `nobleContractAddress` could be the same or different. From codecider:

```
nobleContractAddress: Hex;
- The address of Noble's wrapper contract that users interact with via NobleExpress
- msg.sender of DepositAndBurn must be this address to prevent users from calling replaceDepositForBurn
```

I'm not sure where exactly we'd find the noble wrapper contract addresses (if they exist). Might need to ask those folks directly.

Reply:
yes, let's add that pointer, @samsiegart 

I wonder how to track down the date the contract was deployed. A date is always a nice thing to have in a citation.

Reply:
See https://github.com/Agoric/agoric-sdk/pull/10679. Feel free to provide suggestions for a better name in PR or offline

Reply:
on 2nd thought, to answer my own question, yes, we did choose this address: this is a test config. If we want to re-deploy the contract at a different address in our test env, we don't have to coordinate with anybody else.